src/gwt/src/org/rstudio/studio/client/application/ui/RStudioThemes.java

```diff
@@ -44,4 +55,38 @@ public static void initializeThemes(String themeName,
    public static boolean isFlat(UIPrefs prefs) {
       return prefs.getFlatTheme().getValue() != "classic"; 
    }
+   
+   private static boolean usesScrollbars() {
+      if (usesScrollbars_ != null) return usesScrollbars_;
+      
+      if (!BrowseCap.isMacintosh()) {
+         usesScrollbars_ = true;
+      }
+      else {
+         Element parent = Document.get().createElement("div");
+         parent.getStyle().setWidth(100, Unit.PX);
+         parent.getStyle().setHeight(100, Unit.PX);
+         parent.getStyle().setOverflow(Overflow.AUTO);
+         parent.getStyle().setVisibility(Visibility.HIDDEN);
+         parent.getStyle().setPosition(Position.ABSOLUTE);
```

Comment:
Yes, my bad, was thinking of setting this to `FIXED` from your previous comment, works well either way but good what we know works best within the IDE.